src/Components/CreateImageWizard/utilities/useValidation.tsx

```diff
@@ -122,6 +132,51 @@ export function useRegistrationValidation(): StepValidation {
     };
   }
 
+  if (registrationType === 'register-satellite') {
+    const errors = {};
+    if (caCertificate && (caCertificate === '' || !isValidPEM(caCertificate))) {
+      Object.assign(errors, {
+        certificate:
+          'Valid certificate must be present if you are registering Satellite.',
+      });
+    }
+    if (registrationCommand === '' || !registrationCommand) {
+      Object.assign(errors, {
+        command: 'No registration command for Satellite registration',
+      });
+    }
+    try {
+      const match = registrationCommand?.match(
+        /Bearer\s+([\w-]+\.[\w-]+\.[\w-]+)/
+      );
+      if (!match) {
+        Object.assign(errors, { command: 'Invalid token' });
```

Comment:
This can also be cause by the token not being there right?
```suggestion
        Object.assign(errors, { command: 'Invalid or missing token' });
```